pom.xml

```diff
@@ -319,6 +319,7 @@
               <include>org.glassfish.**</include>
               <include>com.github.jnr:*</include>
               <include>org.ow2.asm:*</include>
+              <include>com.google.guava:**</include>
```

Comment:
I believe this will include the original Guava classes as-is in the shaded jar.

I tested it out and:

```
dxia/relocate-guava!docker-client $> unzip -l target/docker-client-8.8.3-SNAPSHOT-shaded.jar | grep GwtCompatible
      640  08-05-2017 12:48   com/google/common/annotations/GwtCompatible.class
```

so I think this line should be removed (although maybe this behavior changes if the relocation/pattern below is corrected?)